annotations/build.gradle

```diff
@@ -1,6 +1,2 @@
 apply plugin: 'org.jetbrains.kotlin.jvm'
-apply plugin: 'com.vanniktech.maven.publish'
-
-dependencies {
-  implementation deps.kotlin.stdlib
```

Comment:
Oh yeah, this is automatically added by the `org.jetbrains.kotlin.jvm` plugin now, right? Cool!